aten/src/THC/THCOffsetInfo.cuh

```diff
@@ -0,0 +1,89 @@
+#ifndef THC_OFFSET_INFO_INC
```

Comment:
Can you move this into TensorInfo.cuh so it exists with the other offset code?

Reply:
Done.